### DISCUSSION TITLE
considering-asp-net-mvc-ui-controls-learn-html-and-javascript/

### DISCUSSION THREAD

**Discussion by martijnboland** [martijnboland/blog#33] [Blog comments]
# considering-asp-net-mvc-ui-controls-learn-html-and-javascript/

https://blogs.taiga.nl/martijn/posts/considering-asp-net-mvc-ui-controls-learn-html-and-javascript/

**Comment by disqus-comments-migrator** [comment]
<em>Joel Abrahamsson commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at augustus 13 2009, 06:22</em>

---

<p>Good post and spot on!</p>

**Comment by disqus-comments-migrator** [comment]
<em>Ben Anderson commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at augustus 13 2009, 07:26</em>

---

<p>Couldn't agree more! I was initially exited when I heard there was some jQuery *helpers* from Telerik, but they are really just an unnecessary abstraction so devs don't need to learn JS. I can't see any other real benefit from using them.</p>

**Comment by disqus-comments-migrator** [comment]
<em>Dmitriy Nagirnyak commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at augustus 14 2009, 11:18</em>

---

<p>Cannot agree more!<br>In fact all the 3rd party controls bring additional complexity, dependency,  maintenance, deployment problems just to state some.</p>

**Comment by disqus-comments-migrator** [comment]
<em>Tim Jones commented [on Disqus](https://disqus.com/home/forum/martijnboland/) at augustus 23 2009, 09:49</em>

---

<p>I agree with you about the toolkits you linked to, but I think Coolite (<a href="http://www.coolite.com/)" rel="nofollow noopener" title="http://www.coolite.com/)">http://www.coolite.com/)</a> might be on to something. Their toolkit (actually just a wrapper on to ExtJS) forgoes the WebForms postback model, and in the process opens itself up to usage in both WebForms and MVC scenarios.</p>
